web/src/lazy/zxcvbn.js

```diff
@@ -0,0 +1,9 @@
+let zxcvbn = () => ({ score: 0 });
+
+export default (...args) => {
+  import(/* webpackChunkName: "zxcvbn" */ 'zxcvbn').then(real => {
```

Comment:
Setting `/* webpackChunkName: "whatever" */` is a hint to webpack that this should be a separate file.